sync/windows/updates.ps1

```diff
@@ -0,0 +1,23 @@
+<#
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#>
+Write-Output "##########################"
+Write-Output "Installing Windows Updates"
+
+$Updates = Start-WUScan
+Write-Host "Updates Found: " $Updates.Count
+Install-WUUpdates -Updates $Updates
```

Comment:
This process is taking a few minutes.

I will review the box rebuild and add this option.
Newer version of containerd, and probably newer binaries by default.